mappings/net/minecraft/server/SaveLoading.mapping

```diff
@@ -0,0 +1,35 @@
+CLASS net/minecraft/class_7237 net/minecraft/server/SaveLoading
+	METHOD method_42096 (Lnet/minecraft/class_6860;Lnet/minecraft/class_5350;Ljava/lang/Throwable;)V
+		ARG 1 dataPackContents
+		ARG 2 throwable
+	METHOD method_42097 (Lnet/minecraft/class_5455$class_6890;Lnet/minecraft/class_7237$class_7239;Lnet/minecraft/class_6860;Ljava/lang/Object;Lnet/minecraft/class_5350;)Ljava/lang/Object;
+		ARG 4 dataPackContents
+	METHOD method_42098 load (Lnet/minecraft/class_7237$class_6906;Lnet/minecraft/class_7237$class_6907;Lnet/minecraft/class_7237$class_7239;Ljava/util/concurrent/Executor;Ljava/util/concurrent/Executor;)Ljava/util/concurrent/CompletableFuture;
+		ARG 0 functionLoaderConfig
+		ARG 1 savePropertiesSupplier
+		ARG 2 saveUserFactory
+		ARG 3 prepareExecutor
+		ARG 4 applyExecutor
+	CLASS class_6906 FunctionLoaderConfig
+		FIELD comp_361 commandEnvironment Lnet/minecraft/class_2170$class_5364;
+		FIELD comp_362 functionPermissionLevel I
+		FIELD comp_634 dataPacks Lnet/minecraft/class_7237$class_7238;
+		METHOD comp_361 commandEnvironment ()Lnet/minecraft/class_2170$class_5364;
+		METHOD comp_362 functionPermissionLevel ()I
+		METHOD comp_634 dataPacks ()Lnet/minecraft/class_7237$class_7238;
+	CLASS class_6907 SavePropertiesSupplier
+		METHOD get (Lnet/minecraft/class_3300;Lnet/minecraft/class_5359;)Lcom/mojang/datafixers/util/Pair;
+			ARG 1 resourceManager
+			ARG 2 dataPackSettings
+	CLASS class_7238 DataPacks
+		FIELD comp_635 manager Lnet/minecraft/class_3283;
+		FIELD comp_636 settings Lnet/minecraft/class_5359;
+		METHOD comp_635 manager ()Lnet/minecraft/class_3283;
+		METHOD comp_636 settings ()Lnet/minecraft/class_5359;
+		METHOD method_42099 load ()Lcom/mojang/datafixers/util/Pair;
+	CLASS class_7239 SaveUserFactory
```

Comment:
Can we call this `Applier` as "user" is somewhat reminiscent of the offline players?

Reply:
Sounds good